### PR TITLE
Docs: edit export alerting resources feature in Cloud what's new

### PR DESCRIPTION
**Pull request by imatwawana**

### CODE REVIEW

**Comment by brendamuir** [comment]
provisioning - 16th october (or last week). confirming the rest.

**Comment by brendamuir** [comment]
we can delete recovery thresholds from 10.2 and from cloud -- it's not going out with 10.2 and hasn't been released to cloud yet.

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -18,6 +18,28 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Edit file-provisioned alerting resources
+
+<!-- George Robinson -->
+<!-- OSS, Enterprise -->
+<Release date>
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:25]:
```suggestion
October 16, 2023
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -18,6 +18,28 @@ weight: -37
...
+
+_Generally available in all editions of Grafana_
+
+To reduce the noise of flapping alerts, you can set a recovery threshold different from the alert threshold.
+
+Flapping alerts occur when a metric hovers around the alert threshold condition and they may lead to frequent state changes, resulting in too many notifications being generated.
+
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:42]:
```suggestion
```

**Review by brendamuir — APPROVED** [review]

**Pull request merged**